projects/pokemon_game.py

```diff
@@ -28,41 +29,36 @@ def __init__(self, name, primary_type, max_hp, hp) -> None:
     def battle (self, computer_pokemon):
```

Comment:
**issue**:  You are using the number `5` as what is sometimes referred to as a "magic number". Basically, it's a number that doesn't really have much meaning by itself, but does fit in the context of your app as a whole. In this case, it seems like you chose `5` because if you start with hp at `50`, it's more balanced for your hp to go down 5 instead of 1.

So, I suggest (and it's generally Python good practice) moving numbers like these to become constants at the top level of your script, under the imports:

```py
import random

HP_LOSS_DECREMENT = 5
HP_FEED_INCREMENT = 1

class Pokemon(): ...
```

Then use these in your script. This also has the advantage of making the behavior of your app much easier to customize.